package.json

```diff
@@ -28,6 +28,7 @@
   "dependencies": {
     "aws-sdk": "^2.1148.0",
     "axios": "^0.27.2",
+    "inquirer": "^8.2.4",
```

Comment:
Since `inquirer` isn't used by the lambda and is only used by a developer from their local machine, let's move it to `devDependencies` 🙂.

I typically don't move it manually. I let `npm` handle the updates in case it needs to update `package-lock.json` along the way:
```
npm uninstall inquirer
npm install inquirer --save-dev
```